### ISSUE TITLE
Instructions Clarification

### ISSUE THREAD

**Issue by jessemorton** [nawarkhede/getdrip#2]
It might help others to clarify that for the token being passed to GetDripAPI, it's necessary to go through the OAuth steps on the [Drip site](https://www.getdrip.com/docs/rest-api#authentication). At first I was trying to use my non-OAuth personal API token, which results in not-allowed responses from Drip.

Thanks for the tool!

**Comment by ktbyers** [comment]
@jessemorton Yes, I just did the same thing...took me a while to work out that I needed to convert over API token and not oath.

@nawarkhede Are you open to pull requests on this, if I extend what you have here to support either an API token or OAuth?

**Comment by nawarkhede** [comment]
@ktbyers yes I am open to pull requests on this. It will be great if we support both API token as well as OAuth. Let me know your thoughts on this.

**Comment by ktbyers** [comment]
@nawarkhede Okay, I will probably try to submit a PR on this. Let me double check my code and make sure I didn't break anything from your original oauth solution.

**Comment by bwesen** [comment]
Hi, was this ever pushed? The obvious use of this lib is to just paste your API Token as, indeed, the doc says "Token". Most other cloud SDKs expect this as well.

**Comment by ktbyers** [comment]
FYI, I never upstreamed this, but I did implement token support in my fork:

https://github.com/ktbyers/getdrip/blob/master/getdrip/__init__.py#L31

**Comment by bwesen** [comment]
> FYI, I never upstreamed this, but I did implement token support in my fork:
> 
> https://github.com/ktbyers/getdrip/blob/master/getdrip/__init__.py#L31

Actually I just tried the HEAD on this repo and it seems it works with the normal API tokens. It's just too bad it never was pushed to PyPi.
